src/renderers/dom/stack/client/ReactDOMComponent.js

```diff
@@ -64,10 +64,15 @@ function getDeclarationErrorAddendum(internalInstance) {
   if (internalInstance) {
     var owner = internalInstance._currentElement._owner || null;
     if (owner) {
-      var name = owner.getName();
-      if (name) {
-        return ' This DOM node was rendered by `' + name + '`.';
+      if (owner.getName) {
+        var name = owner.getName();
+        if (name) {
+          return ' This DOM node was rendered by `' + name + '`.';
+        }
+      } else {
+        return ' This DOM node was rendered by `' + owner.tag + '`.';
```

Comment:
This component is only used by Stack so you don't need to handle Fiber components; the Fiber equivalent is `ReactDOMFiberComponent`. (If you did, `tag` wouldn't be the right attribute; it's just a constant/number. [`getComponentName`](https://github.com/facebook/react/blob/master/src/shared/utils/getComponentName.js) can be used to log the display name of stack-or-fiber components FWIW.) :smile: